### ISSUE TITLE
 Upload files to phone storage

### ISSUE THREAD

**Issue by lin54241930** [DeviceFarmer/stf#95]
**Is your feature request related to a problem? Please describe.**
When I want to upload a file to the phone storage, I can only upload it through some other methods, and I cannot directly upload the file to the phone storage through STF.
**Describe the solution you'd like**
I hope there is a way to upload files to phone storage like installing APK package
**Describe alternatives you've considered**
I tried to upload apk to achieve this function, but I failed to do it, I hope someone can achieve this function

**Comment by denis99999** [comment]
@lin54241930 , to upload an APK you have just to drap & drop the file into the `Dashbord->App Upload` tab of the controlling device

**Comment by koral--** [comment]
I think the OP wants to upload files other than APK.

**Comment by lin54241930** [comment]
> I think the OP wants to upload files other than APK.
yes!  I want to upload some other files to the phone storage

**Comment by denis99999** [comment]
You can do that using `adb push` command, but if I well understand you would like to do that through the UI ?

**Comment by lin54241930** [comment]
> You can do that using `adb push` command, but if I well understand you would like to do that through the UI ?

Yes, I hope there is a separate UI module that can upload files to the specified directory of the phone, similar to a module like installing APK
